app/javascript/api/types/rails/listings/RailsTranslation.ts

```diff
@@ -0,0 +1,15 @@
+export enum RailsTranslationLanguage {
+  EN = "EN",
+  ES = "ES",
+  ZH = "ZH",
+  TL = "TL",
+}
```

Comment:
nit: Add a space between declarations. Also, strange that our linter/formatter doesn't catch this.